src/v2i-hub/TimPlugin/src/VehicleLocate.cpp

```diff
@@ -0,0 +1,196 @@
+/*
+ * VehicleLocate.cpp
```

Comment:
This file may be deleted if we don't need to handle vehicle location.

Reply:
Addressed